src/commands/build.command.ts

```diff
@@ -156,8 +156,32 @@ export function buildCommand({project, verbose, clean, local, main, watch, skipB
 }
 
 export function buildTsRun(cli) {
-  const {project, watch, verbose, clean, local, skipBundles} = cli.flags;
-  let main = cli.flags.main || 'index.ts';
+  const config = cli.flags.config ? require(path.resolve(cli.flags.config)) : {};
+  let {src, entryPoints, watch, verbose, clean, local, skipBundles} = config;
+  const main = cli.flags.main || 'index.ts';
+  const project = cli.flags.project || src;
+  verbose = cli.flags.verbose || verbose;
+  watch = cli.flags.watch || watch;
+  clean = cli.flags.clean || clean;
+  skipBundles = cli.flags.skipBundles || skipBundles;
+
+  if (!project && (!entryPoints || entryPoints.length)) {
```

Comment:
Entry point is a relative path to file, not folder

Reply:
So there are should be an array of folders and each folder should have at least one entry point, like ng cli has